diff --git a/node_modules/body-parser/lib/types/json.js b/node_modules/body-parser/lib/types/json.js
index 15c54bb..90f8507 100644
--- a/node_modules/body-parser/lib/types/json.js
+++ b/node_modules/body-parser/lib/types/json.js
@@ -15,6 +15,7 @@
 var debug = require('debug')('body-parser:json')
 var read = require('../read')
 var { normalizeOptions } = require('../utils')
+var JSONbig = require("json-bigint");
 
 /**
  * Module exports.
@@ -71,7 +72,7 @@ function json (options) {
 
     try {
       debug('parse json')
-      return JSON.parse(body, reviver)
+      return JSONbig.parse(body, reviver)
     } catch (e) {
       throw normalizeJsonSyntaxError(e, {
         message: e.message,
@@ -113,7 +114,7 @@ function createStrictSyntaxError (str, char) {
   }
 
   try {
-    JSON.parse(partial); /* istanbul ignore next */ throw new SyntaxError('strict violation')
+    JSONbig.parse(partial); /* istanbul ignore next */ throw new SyntaxError('strict violation')
   } catch (e) {
     return normalizeJsonSyntaxError(e, {
       message: e.message.replace(JSON_SYNTAX_REGEXP, function (placeholder) {
@@ -159,7 +160,7 @@ function normalizeJsonSyntaxError (error, obj) {
   }
 
   // replace stack before message for Node.js 0.10 and below
-  error.stack = obj.stack.replace(error.message, obj.message)
+  error.stack = obj.stack?.replace(error.message, obj.message)
   error.message = obj.message
 
   return error
