diff --git a/node_modules/typescript-json-schema/dist/typescript-json-schema.js b/node_modules/typescript-json-schema/dist/typescript-json-schema.js
index 3eab401..69658a4 100644
--- a/node_modules/typescript-json-schema/dist/typescript-json-schema.js
+++ b/node_modules/typescript-json-schema/dist/typescript-json-schema.js
@@ -470,6 +470,9 @@ var JsonSchemaGenerator = (function () {
             else if (flags & ts.TypeFlags.Boolean) {
                 definition.type = "boolean";
             }
+			else if (flags & ts.TypeFlags.BigInt) {
+				definition.type = "bigint";
+			}
             else if (flags & ts.TypeFlags.ESSymbol) {
                 definition.type = "object";
             }
