diff --git a/node_modules/typescript-json-schema/dist/typescript-json-schema.js b/node_modules/typescript-json-schema/dist/typescript-json-schema.js
index 47e1598..8397b9d 100644
--- a/node_modules/typescript-json-schema/dist/typescript-json-schema.js
+++ b/node_modules/typescript-json-schema/dist/typescript-json-schema.js
@@ -432,6 +432,9 @@ var JsonSchemaGenerator = (function () {
             else if (flags & ts.TypeFlags.Boolean) {
                 definition.type = "boolean";
             }
+            else if (flags & ts.TypeFlags.BigInt) {
+                definition.type = "bigint";
+            }
             else if (flags & ts.TypeFlags.Null) {
                 definition.type = "null";
             }
