diff --git a/Spacebar.Models.Db/Contexts/SpacebarDbContext.cs b/Spacebar.Models.Db/Contexts/SpacebarDbContext.cs
index 48bd953b..831bddce 100644
--- a/Spacebar.Models.Db/Contexts/SpacebarDbContext.cs
+++ b/Spacebar.Models.Db/Contexts/SpacebarDbContext.cs
@@ -1,7 +1,6 @@
-using System;
-using System.Collections.Generic;
 using Microsoft.EntityFrameworkCore;
 using Spacebar.Models.Db.Models;
+using Stream = Spacebar.Models.Db.Models.Stream;

 namespace Spacebar.Models.Db.Contexts;

