--- Spacebar.Db/Models/User.cs.orig	2025-10-05 22:04:37.168566856 +0200
+++ Spacebar.Db/Models/User.cs	2025-10-05 22:07:11.519980808 +0200
@@ -92,8 +92,8 @@
     [Column("email", TypeName = "character varying")]
     public string? Email { get; set; }
 
-    [Column("flags")]
-    public int Flags { get; set; }
+    [Column("flags", TypeName = "character varying")]
+    public string Flags { get; set; }
 
     [Column("public_flags")]
     public int PublicFlags { get; set; }
